app/controllers/desks_controller.rb

```diff
@@ -23,6 +23,9 @@ def show
 
   def update
     if @desk.update(desk_params)
+      if broadcast_needed?
+        ActionCable.server.broadcast('desks', @desk.slice('id', 'name', 'occupied'))
```

Comment:
Nit: I'd probably wrap the event broadcast in a method so that you can do 
```ruby
broadcast_updated_desk_event if broadcast_needed?
```

Reply:
I like it